library/rendering/entity_rendering/src/testmod/java/org/quiltmc/qsl/rendering/entity/test/client/ClientEntityRenderingTestmod.java

```diff
@@ -38,7 +38,7 @@ public void onInitializeClient(ModContainer mod) {
 		ArmorRenderingRegistry.registerTextureProvider(this, EntityRenderingTestmod.QUILT_LEGGINGS);
 	}
 
-	private static final Identifier LEGGINGS_TEXTURE_ID = EntityRenderingTestmod.id("candypants.png");
+	private static final Identifier LEGGINGS_TEXTURE_ID = EntityRenderingTestmod.id("pants_made_out_of_candied_quilts.png");
```

Comment:
You missed the point here; I wanted to avoid a food mention here (see: the announcement on Quiltcomm that they still have to write)

Reply:
Whoops! Lemme fix that rq